posts/2014-11-14-ecosystem-vulnerability-tools.md

```diff
@@ -0,0 +1,333 @@
+* [Environmental Sensitivity Index](#environmental-sensitivity-index): Provides information on at-risk coastal resources, sensitive shorelines, and human-use resources (i.e., public beaches and parks) to assist in the aftermath of oil spills.
+* [NOAA’s Coral Reef Information System (CoRIS)](#noaa's-coral-reef-information-system-(coris)): A web-based information portal that provides access to NOAA coral reef information and data products with emphasis on the U.S. states, territories and remote island areas.
+* [Ocean Biogeographic Information System (OBIS)](#ocean-biogeographic-information-system-(obis)): A repository of data related to marine species.
+
+## Water Resources and Drought
+
+* [National Integrated Drought Information System (NIDIS) Drought Portal](#national-integrated-drought-information-system-(nidis)-drought-portal): Provides a series of tools and maps that show real-time or near-time information about drought across the U.S.
+* [Paleoclimatology Software Tools](#paleoclimatology-software-tools): A collection of paleoclimatological datasets and suggestions for software packages that can be used to analyze the data.
+* [Soil Climate Analysis Network (SCAN)](#soil-climate-analysis-network-(scan)): SCAN integrates information from soil-climate data stations around the United States, focusing on the agricultural areas.
+
+## Wildfires
+
+* [Landfire](#landfire): A program that provides over 20 national geo-spatial layers (e.g. vegetation, fuel, disturbance, etc.), databases, and ecological models that are available to the public for the US and insular areas.
+
+
+### Integrated Taxonomic Informatin System (IT IS)
+**Sponsor:** USGS
+**Name:** Integrated Taxonomic Informatin System (IT IS)
+**URL:** <http://www.itis.gov/tools.html>
+**Summary:**
+The ITIS database currently provides tools to compare species names with their hierarchical classifications. Agencies that typically cannot afford to maintain taxonomic data will have access to high quality taxonomic information through ITIS.
+**How it works:**
+ITIS currently includes two tools. The first is a prototype to compare taxonomy and nomenclature. The Taxonomic Workbench is a Windows-based software tool used for editing and manipulating taxonomic data for submission into the ITIS online system. This software is used for entering new data into the ITIS database, as well as updating records already in the online system.
+**Application:**
+Federal agencies have the ability to use the ITIS tool in order to promote improved coordination of efforts among federal agencies, thereby increasing productivity and saving resources.
+**Preview:**
+[![](https://s3-us-gov-west-1.amazonaws.com/cg-0817d6e3-93c4-4de8-8b32-da6919464e61/IntegratedTaxonomicInformatinSystem.png)](http://www.itis.gov/tools.html)
```

Comment:
I think this is failing cause it's missing an [alt text](https://www.markdownguide.org/basic-syntax/#images-1) in the `[]` here.

Reply:
fixed